solidity/contracts/system/TBTCSystem.sol

```diff
@@ -468,6 +473,27 @@ contract TBTCSystem is Ownable, ITBTCSystem, DepositLog {
         emit EthBtcPriceFeedAdded(address(_nextEthBtcPriceFeed));
     }
 
+    /// @notice Locks versions of Keep factories. When lock is set vendor contracts
+    /// won't be called anymore to obtain the latest Keep factories versions.
+    /// It requires expected factories addresses to be provided to protect from
+    /// locking on unexpected addresses.
+    /// @param _expectedKeepStakeFactory Expected KEEP-staked factory address
+    /// @param _expectedFullyBackedFactory Expected ETH-staked factory address
```

Comment:
There is no ETH-staked factory, we should say about ETH-bond-only or fully backed.

Reply:
b35547358432c1031368e630f51602fd1146d732